roles/etcd/defaults/main.yaml

```diff
@@ -14,12 +13,12 @@ etcd_peer_cert_file: "{{ etcd_conf_dir }}/peer.crt"
 etcd_peer_key_file: "{{ etcd_conf_dir }}/peer.key"
 
 etcd_initial_cluster_state: new
-etcd_initial_cluster_token: etcd-cluster-1
+etcd_initial_cluster_token: etcd-os-cluster-1
```

Comment:
I know there are places where this isn't currently the case, but I'd like to keep openshift specifics outside of the etcd role as much as possible, ideally merging a redhat/centos/fedora compatible role back into https://github.com/retr0h/ansible-etcd when time allows.

Perhaps set the token in the playbook around about https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-etcd/config.yml#L71